Broken fzf test

```
Error:  Uncaught exception in test: Vim(call):[Fzf-lua] fn_selected threw an error: [string ":lua"]:93: attempt to index field '__resume_data' (a nil value) at command line..function RunTest[2]..Test[15]..Test_MultipleSelection, line 13
Error: Process completed with exit code 1.
```

https://github.com/kaihowl/dotfiles/actions/runs/7716686749/job/21034081603?pr=736#step:5:3404

--- a/nvim/config--nvim--init.vim.symlink
+++ b/nvim/config--nvim--init.vim.symlink
@@ -356,7 +356,7 @@ function git_set_qf(selected, opts)
   vim.fn.setqflist({}, " ", {
     nr = "$",
     items = converted,
-    title = opts.__resume_data.last_query,
+    title = require('fzf-lua').get_last_query()
   })
   vim.cmd("copen")
 end

